src/pages/schedule-page.html

```diff
@@ -1,24 +1,94 @@
 <link rel="import" href="../../bower_components/polymer/polymer-element.html">
 
+<link rel="import" href="../../bower_components/app-route/app-route.html">
+<link rel="import" href="../../bower_components/iron-pages/iron-pages.html">
+
+<link rel="import" href="../mixins/utils-functions.html">
+<link rel="import" href="../mixins/redux-mixin.html">
+<link rel="import" href="../elements/shared-styles.html">
+<link rel="import" href="../elements/schedule-day.html">
+
 <dom-module id="schedule-page">
   <template>
     <style is="custom-style" include="shared-styles flex flex-alignment"></style>
 
     <style>
 
-      :host  {
+      :host {
         display: block;
       }
 
+      @media (max-width: 640px) {
+        .container {
+          padding: 0 0 32px;
+        }
+      }
+
+      @media (min-width: 640px) {
+        :host {
+          background-color: #fff;
+        }
+      }
+
     </style>
 
-   <h3>Hello from the schedule page</h3>
+    <app-route
+      route="[[route]]"
+      pattern="/:day"
+      data="{{routeData}}"
+      query-params="{{queryParams}}"
+    ></app-route>
+
+    <div class="container">
+      <iron-pages selected="[[subRoute]]" attr-for-selected="name">
+        <template is="dom-repeat" items="[[transformToArray(schedule.days, 'date')]]" as="day">
+          <schedule-day
+            name$="[[day.date]]"
+            day="[[day]]"
```

Comment:
no, we don't need to have this information as an attribute, just passing the object to the children